src/smpi/include/private.h

```diff
@@ -60,7 +60,7 @@ typedef SMPI_Graph_topology *MPIR_Graph_Topology;
 
 typedef SMPI_Dist_Graph_topology *MPIR_Dist_Graph_Topology;
 
-XBT_PRIVATE SMPI_Process* smpi_process();
+XBT_PUBLIC(SMPI_Process*) smpi_process();
```

Comment:
Make sure these private to public changes are really necessary; it's pretty ugly.